functions/backup.bash

```diff
@@ -154,75 +138,69 @@ create_backup_config() {
 }
 
 amanda_setup() {
+  local matched
```

Comment:
Remove changes from previous iteration of this PR that are no longer necessary.

Reply:
huh? "matched" is used or what do you mean ?

Reply:
I was referring to the function in general, in a sense from looking at the comment in `openhabian-setup.sh`

Reply:
be more specific please what would need to be removed?

Reply:
Never-mind, it was confusion because of other changes.